templates/terraform/examples/base_configs/test_file.go.erb

```diff
@@ -58,10 +58,14 @@ func TestAcc<%= test_slug -%>(t *testing.T) {
 	vcrTest(t, resource.TestCase{
 		PreCheck:     func() { testAccPreCheck(t) },
 		<% unless versioned_provider -%>
-		Providers:    testAccProviders,
+		Providers: testAccProviders,
 		<% else -%>
-		Providers:    testAccProvidersOiCS,
+		Providers: testAccProvidersOiCS,
 		<% end -%>
+		<%#- The Terraform test runner doesn't let us import resources with aliased providers (yet) -%>
```

Comment:
I think this comment can be removed? I'm not sure what it means in this context, at least.